### ISSUE TITLE
trouble with arloader

### ISSUE THREAD

**Issue by CalebEverett** [robbie0202/upload-nfts-new-img#1]
Hi, are you still having trouble uploading with arloader?

**Comment by robbie0202** [comment]
Yes when I am attempting to upload a folder with all my files I keep
getting this error, If I try to do a single file I have no issue

what am i doing wrong?

**Comment by CalebEverett** [comment]
@robbie0202 can you let me know what version you are using, the command you are entering and the error you are getting?

**Comment by CalebEverett** [comment]
I'm running the `upload_nfts` example here, which effectively uploads files from a folder twice, once for pngs and again for the metadata jsons, without any issues.
